index.html

```diff
@@ -10320,8 +10320,9 @@ <h4>Example Attribute Usage</h4>
 		</section>
 	</section>
 	<section>
-		<h2>Translatable States and Properties</h2>
-		<p>The HTML specification states that other specifications can define <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a>. In order to be understandable by assistive technology users, the values of the following <a>states</a> and <a>properties</a> are <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a> and should be translated when a page is localized:</p>
+		<h2>Translatable Attributes</h2>
+		<p>The HTML specification states that other specifications can define <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a>. The source translation language of each value is defined by use of <a href="https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes">the <code>lang</code> attribute</a> on the element or its ancestor elements.</p>
```

Comment:
Maybe say "attribute value"? When it just says "value", I'm thinking, "what value?", "value of what?".

```suggestion
		<p>The HTML specification states that other specifications can define <a href="https://html.spec.whatwg.org/multipage/dom.html#translatable-attributes">translatable attributes</a>. The source translation language of each attribute value is defined by use of <a href="https://html.spec.whatwg.org/multipage/dom.html#the-lang-and-xml:lang-attributes">the <code>lang</code> attribute</a> on the element or its ancestor elements.</p>
```